usr/src/cmd/slim-install/svc/media-fs-root

```diff
@@ -285,9 +285,15 @@ elif [ $LIVECD = 1 ]; then
 	livemode=`$PRTCONF -v|$SED -n '/livemode/{;n;p;}'|$CUT -f 2 -d\'`
 	if [ "$livemode" = "text" ]; then
 		/usr/sbin/svcadm disable -t application/graphical-login/gdm
+		/usr/sbin/svcadm disable -t application/graphical-login/lightdm
 	else
 		[ "$livemode" = "vesa" ] && cp /.cdrom/jack/xorg.conf.vesa /etc/X11/xorg.conf
-		/usr/sbin/svcadm enable application/graphical-login/gdm
+		if [ -f /usr/sbin/lightdm ]; then
+			[ -f /usr/sbin/gdm ] && /usr/sbin/svcadm disable application/graphical-login/gdm
```

Comment:
It should be enabled by default, if exists, as we do it in profile for live image.